src/components/sbb-image/sbb-image.e2e.ts

```diff
@@ -1,39 +1,38 @@
+import { assert, expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { SbbImage } from './sbb-image';
 import images from '../../global/images';
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 
 describe('sbb-image', () => {
-  let element: E2EElement, page: E2EPage;
+  let element: SbbImage;
 
   it('renders', async () => {
     const url = images[0];
+    element = await fixture(html`<sbb-image image-src="${url}"></sbb-image>`);
 
-    page = await newE2EPage();
-    await page.setContent(`<sbb-image image-src="${url}"></sbb-image>`);
-    element = await page.find('sbb-image');
-    expect(element).toHaveClass('hydrated');
-
-    await page.waitForChanges();
+    assert.instanceOf(element, SbbImage);
+    await element.updateComplete;
     // Wait five seconds in hope the image will successfully be loaded
     // TODO: Find more reliable solution
-    await new Promise((res) => setTimeout(res, 5000));
-    await page.waitForChanges();
+    await new Promise((res) => setTimeout(res, 2000));
+    await element.updateComplete;
```

Comment:
There is an utility command that lets you wait for a timeout
```suggestion
    await aTimeout(2000)
```